### ISSUE TITLE
Some Intl options don't work as expected on iOS

### ISSUE THREAD

**Issue by michalzuk** [facebook/hermes#812]
## Bug Description

`Intl.NumberFormat` is not working properly. Seems to ignore a few of possible options. In my case, tested ignored properties are `minimumFractionDigits` and `maximumFractionDigits`

- [x] I have run `gradle clean` and confirmed this bug does not occur with JSC

Hermes version:
React Native version (if any): 0.70
OS version (if any): iOS 15.5
Platform (most likely one of arm64-v8a, armeabi-v7a, x86, x86_64):

## Steps To Reproduce

1. Build a clean RN project (0.70 uses Hermes by default)
2. Add basic usage of `Intl.NumberFormat` usage and provide `minimumFractionDigits` and `miximumFractionDigits`.
3. Check displayed/logged result

code example:
```
import React from 'react';
import {StyleSheet, Text, View} from 'react-native';

const value = Intl.NumberFormat('en-US', {
  minimumFractionDigits: 1,
  maximumFractionDigits: 1,
}).format(10101019019.9090901);

const App = () => {
  return (
    <View style={styles.container}>
      <Text>{value}</Text>
    </View>
  );
};

const styles = StyleSheet.create({
  container: {
    flex: 1,
    backgroundColor: '#fff',
    justifyContent: 'center',
    alignItems: 'center',
  },
});

export default App;
```

Result on iOS:
![image](https://user-images.githubusercontent.com/18529692/189097801-0a795f80-ee26-4c17-9911-f1f0206c20be.png)

Result on Android:
![image](https://user-images.githubusercontent.com/18529692/189097461-46de17d8-29ed-48a7-9cfe-0e78755197f7.png)

## The Expected Behavior
Formatted value should be displayed in the same way as on Android.

**Comment by tomasmozeris** [comment]
Having same issue on iOS for `maximumFractionDigits`

**Comment by neildhar** [comment]
Thanks for reporting this, this has actually already been fixed in 42fe3572f86c2d8d9648cb5e8414b487cb45931f. If it is a major issue, please request a pick for the next minor release of RN. You can do so here: https://github.com/reactwg/react-native-releases/discussions/32.
